### PR TITLE
lint github actions

### PR DESCRIPTION
**Pull request by yokomotod**
Thanks to #11 now our code will pass the linting, but it is still not checked by CI, so I fixed it.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -23,7 +23,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install tox-gh-actions poetry
+        python -m pip install poetry
         poetry install
+        poetry run python -m pip install tox-gh-actions
```
Comment (yokomotod) [inline .github/workflows/test.yml:28]: Another fixing:

previously the `tox-gh-actions` plugin is not recognized and all tox environments are running

ex. last #11 PR's checks: https://github.com/m3dev/kannon/actions/runs/4649331048/jobs/8227600026?pr=11

```
Run poetry run tox

py38: SKIP ⚠ in 0.88 seconds
py39: commands[0]> coverage run -m unittest discover -s test
py39: OK ✔ in 6.63 seconds
py310: commands[0]> coverage run -m unittest discover -s test
py310: OK ✔ in 2.82 seconds
py311: SKIP ⚠ in 0.01 seconds
pyyapf: commands[0]> coverage run -m unittest discover -s test
pyyapf: OK ✔ in 2.68 seconds
pyisort: commands[0]> coverage run -m unittest discover -s test
pyisort: OK ✔ in 2.68 seconds
```

now: https://github.com/m3dev/kannon/actions/runs/4668051726/jobs/8264603602?pr=13

```
Run poetry run tox
py39: freeze> python -m pip freeze --all
py39: pip==23.0.1,setuptools==67.4.0,wheel==0.38.4
tox: py39
  py39: OK (8.03=setup[1.21]+cmd[6.82] seconds)
  congratulations :) (8.46 seconds)
```

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{38,39,310,311,yapf,isort,flake8}
+envlist = py{38,39,310,311},yapf,isort,flake8
```
Comment (yokomotod) [inline tox.ini:2]: additional fix 2:

previous envlist resulsts to `pyyapf, pyisort, pyflake8` so `[testenv:yapf]` setting is not used and run default `coverage run -m unittest discover -s test`.

```
$ poetry run tox
py38: commands[0]> coverage run -m unittest discover -s test
py38: OK ✔ in 1.65 seconds
py39: commands[0]> coverage run -m unittest discover -s test
py39: OK ✔ in 1.44 seconds
py310: commands[0]> coverage run -m unittest discover -s test
py310: OK ✔ in 1.44 seconds
py311: commands[0]> coverage run -m unittest discover -s test
py311: OK ✔ in 1.4 seconds
pyyapf: commands[0]> coverage run -m unittest discover -s test
pyyapf: OK ✔ in 1.43 seconds
pyisort: commands[0]> coverage run -m unittest discover -s test
pyisort: OK ✔ in 1.43 seconds
pyflake8: commands[0]> coverage run -m unittest discover -s test
```

now:

```
$ poetry run tox
py38: commands[0]> coverage run -m unittest discover -s test
py38: OK ✔ in 3.92 seconds
py39: commands[0]> coverage run -m unittest discover -s test
py39: OK ✔ in 1.46 seconds
py310: commands[0]> coverage run -m unittest discover -s test
py310: OK ✔ in 1.45 seconds
py311: commands[0]> coverage run -m unittest discover -s test
py311: OK ✔ in 1.44 seconds
yapf: commands[0]> yapf -dr .
yapf: OK ✔ in 1.67 seconds
isort: commands[0]> isort -c ./kannon ./test
isort: OK ✔ in 0.39 seconds
flake8: commands[0]> pflake8 ./kannon ./test
```

**Review by kitagry — APPROVED** [review]
LGTM! Thank you!

**Review by maronuu — APPROVED** [review]
Thanks!

**Pull request merged**